_data/projects/alda.yml

```diff
@@ -0,0 +1,12 @@
+name: Alda
+desc: A general-purpose, beginner-friendly music programming language
+site: https://github.com/alda-lang/alda
+tags:
+- music
+- language
+- clojure
+- java
+- zeromq
+upforgrabs:
+  name: low-hanging fruit
```

Comment:
I think our `up for grabs` label may have a slightly different meaning than the purpose of up-for-grabs. We use `up for grabs` to identify issues that are ready for action, but which may or may not be beginner-friendly. `low-hanging fruit` is our label for issue which should be easy™ and are suitable for new contributors.